google/ggrc-core
# GGRC-7538 Refactor tree-widget-containers with relation to mapper

# Issue description

Currently tree-widget-container has a lot of logic related to advanced-search and mapper. This logic can be extracted into some component that will handle advanced-search and mapper state and actions.

# Steps to test the changes

1. Open All object page
2. Check tree-view's search (**filter-input, states dropdown, advanced search**) 

# Solution description

Created new **tree-view-filter** component.
All business logic connected to search in tree-view was moved to new component.

# Sanity checklist




# PR Review checklist
